spring-cloud-task-samples/partitioned-batch-job/README.adoc

```diff
@@ -0,0 +1,20 @@
+= Partitioned Job
+
+An example of the usage of the `DeployerPartitionHandler` and
+`DeployerStepExecutionHandler` to partition a Spring Batch job.
+
+== Requirements:
+
+* Java 7 or Above
+
+== Build:
+
+[source,shell,indent=2]
+----
+$ ./mvnw clean install
+----
+
```

Comment:
Need to add a comment that the user needs to export spring_profiles_active=master before running